### PR TITLE
springboot support: missed codepath in earlier fix to store boot-inf/…

### PR DESCRIPTION
**Pull request by sebratton**
…lib jars with full jar name

### CODE REVIEW

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
fixes #2691

File: dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java [hunk 1]
```diff
@@ -74,7 +74,9 @@ private void thin() throws FileNotFoundException, IOException, NoSuchAlgorithmEx
             Set<String> entryNames = new HashSet<>();
             for (Enumeration<JarEntry> entries = sourceFatJar.entries(); entries.hasMoreElements();) {
                 JarEntry entry = entries.nextElement();
-                if (entryNames.add(entry.getName()) && !JarFile.MANIFEST_NAME.equals(entry.getName()) && !entry.getName().startsWith("org")) { // hack to omit spring boot loader
+                String springBootLoaderClassPath = "org/springframework/boot/loader/";
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java]: Make that a constant.

File: dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java [hunk 2]
```diff
@@ -134,12 +136,14 @@ private void storeLibraryInZip(ZipOutputStream libZip, JarEntry entry, String ha
         // create a complete cache that may have no parent.
         String path = entry.getName();
         try (InputStream is = sourceFatJar.getInputStream(entry)) {
-            if (!hashPrefixes.contains(hashPrefix)) {
-                libZip.putNextEntry(new ZipEntry(hashPrefix));
+            //zip format require trailing '/' for directory entries
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java]: Need to be sure to create entries for the hashPrefix/ folders

**Comment by sebratton** [comment]
#build

**Pull request merged**